repo/meta/schema/v3-resource-schema.json

```diff
@@ -56,7 +63,8 @@
           "additionalProperties": {
             "type": "string"
           },
-          "type": "object"
+          "type": "object",
+          "description": "This is a mapping of aliases to http resources. Aliases enumerated here are available during resolution of marathon.json.mustache file."
```

Comment:
Suggest: `This is a mapping of aliases to HTTP resources. The aliases enumerated here are available during resolution of the marathon.json.mustache file.`